docs/index-developer-guide.md

```diff
@@ -24,11 +24,11 @@ Currently, there are two types of DataMap supported:
 2. MVDataMap: DataMap that leverages Materialized View to accelerate olap style query, like SPJG query (select, predicate, join, groupby). Preaggregate, timeseries and mv DataMap belong to this type of DataMaps.
```

Comment:
Also documents [*.md] files need to be updated for all the changes of this PR
Example, index-developer-guide.md has many datamap words in it still

Reply:
Yes, it will be changed in a new pr.